erigon-lib/chain/chain_config.go

```diff
@@ -82,12 +82,15 @@ type Config struct {
 	Clique *CliqueConfig `json:"clique,omitempty"`
 	Aura   *AuRaConfig   `json:"aura,omitempty"`
 	Bor    *BorConfig    `json:"bor,omitempty"`
+
+	//For not pruning these contracts (e.g. depsoit contract logs needed by CL)
```

Comment:
let's add here more comments. if you know more information now. like: "why CL need it" and "in which mode (archive/non-archive) CL need it" and maybe something else.